packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -101,13 +123,18 @@ export default {
             callback,
             declaredDependenciesNode,
```

Comment:
It's confusing to run `visitFunctionWithDependencies(` without dependencies (`declaredDependenciesNode` can be `undefined` here). I checked, that function seems to correctly bail out if there are none, still confusing.

Reply:
We need to run it because there are other checks before we can bail out. Those other checks require a function body but not a deps node.